view/adminhtml/templates/new_shipment.phtml

```diff
@@ -1,179 +1,253 @@
-    </div>
-    <div class="field choice admin__field admin__field-option field-mypa_return mypa_package-toggle mypa-option-toggle" style="display:none; padding-left: 30px;">
-        <input id="mypa_return"
-               class="admin__control-checkbox"
-               name="mypa_return"
-               value="1"
-               type="checkbox"
-            <?= $block->getDefaultOption('return') == true ? 'checked="checked"' : '';?>
-        />
-        <label class="admin__field-label" for="mypa_return">
-            <span><?=__('Return if no answe') ?></span>
-        </label>
-    </div>
-
-
-    <div class="field choice admin__field admin__field-option field-mypa_large_format mypa_package-toggle mypa-option-toggle" style="display:none; padding-left: 30px;">
-        <input id="mypa_large_format"
-               class="admin__control-checkbox"
-               name="mypa_large_format"
-               value="1"
-               type="checkbox"
-            <?= $block->getDefaultLargeFormat('large_format') == true ? 'checked="checked"' : '';?>
-        />
-        <label class="admin__field-label" for="mypa_return">
-            <span><?=__('Large package') ?></span>
-        </label>
-    </div>
-
-    <div class="field choice admin__field admin__field-option field-mypa_age_check mypa_package-toggle mypa-option-toggle" style="display:none; padding-left: 30px;">
-        <input id="mypa_age_check"
-               class="admin__control-checkbox"
-               name="mypa_age_check"
-               value="1"
-               type="checkbox"
-            <?=$block->getDefaultOptionsWithoutPrice('age_check') == true ? 'checked="checked"' : '';?>
-        />
-        <label class="admin__field-label" for="mypa_age_check">
-            <span><?=__('Age check 18+') ?></span>
-        </label>
-    </div>
-
-    <div class="field choice admin__field admin__field-option field-mypa_label_amount mypa_package-toggle mypa-option-toggle" style="display:none; padding-left: 30px;">
-        <label class="admin__field-label" for="mypa_label_amount" style="display: block">
-            <span><?=__('Label amount:') ?></span>
-        </label>
-        <?php $label_amount = 1; ?>
-        <select name="mypa_label_amount" id="mypa_label_amount"
-                class="required product-custom-option admin__control-select" title="">
+    <div
+        class="field field-mypa_package_type"
+        data-for_mypa_carrier="<?= $carrierName ?>"
+        style="margin: 10px 0 0 30px;">
+        <div class="admin__field-control">
             <?php
-            while ($label_amount <= 10) {
-                echo '<option value="' . $label_amount . '" >' . __($label_amount) . '</option>';
-                $label_amount ++;
-            }
-            ?>
-        </select>
-    </div>
+            $country = $block->getCountry();
+            foreach ($abstractConsignment->getAllowedPackageTypes() as $packageType) {
+                $packageTypeName = $packageType;
```

Comment:
noem hem dan meteen zo in de loop :)

Reply:
Haha oeps, dat was nog door een refactor actie... Aangepast